README.rst

```diff
@@ -146,14 +146,14 @@ apply (the fixer names must be separated by a comma):
 
 .. code-block:: bash
 
-    php php-cs-fixer.phar fix /path/to/dir --rules=linefeed,short_tag,indentation
+    php php-cs-fixer.phar fix /path/to/dir --rules=unix_line_endings,full_opening_tag,no_tabs
 
 You can also blacklist the fixers you don't want by placing a dash in front of the fixer name, if this is more convenient,
 using ``-name_of_fixer``:
 
 .. code-block:: bash
 
-    php php-cs-fixer.phar fix /path/to/dir --rules=-short_tag,-indentation
+    php php-cs-fixer.phar fix /path/to/dir --rules=-full_opening_tag,-no_tabs
```

Comment:
suggestion `no_tabs` -> `no_tab_indentations`, as we do allow tabs for other reasons.